.github/workflows/tests.yaml

```diff
@@ -69,45 +38,45 @@ jobs:
         run: make test
   e2e:
     needs: unit
-    if: "${{ github.event.action == 'labeled' && github.event.label.name == 'tests: run' }}"
```

Comment:
@hessjcg we need to think about how to trigger e2e tests now...?

Reply:
unless we want to leave the label and have it trigger `pull_request` instead of the previous `pull_request_target`